Source/Extension/UIColor+Verse.swift

```diff
@@ -14,49 +14,49 @@ import UIKit
 // a non-optional value to be used across the app
 extension UIColor {
     
-    static let appBackground = UIColor(named: "appBackground") ?? UIColor.white
-    static let mainText = UIColor(named: "mainText") ?? UIColor.black
-    static let secondaryText = UIColor(named: "secondaryText") ?? UIColor.black
-    static let reactionUser = UIColor(named: "reactionUser")!
-    static let primaryAction = UIColor(named: "primaryAction") ?? UIColor.black
-    static let secondaryAction = UIColor(named: "secondaryAction")!
-    static let cardBackground = UIColor(named: "cardBackground")!
-    static let cardBorder = UIColor(named: "cardBorder")!
-    static let splashBackground = UIColor(named: "splashBackgroundColor") ?? UIColor.white
-    static let screenOverlay = UIColor(named: "screenOverlay") ?? UIColor.black.withAlphaComponent(30)
-    static let menuBackgroundColor = UIColor(named: "menuBackgroundColor") ?? UIColor.white
-    static let menuBorderColor = UIColor(named: "menuBorderColor") ?? UIColor.black
-    static let menuSelectedItemBackground = UIColor(named: "menuSelectedItemBackground") ?? UIColor.black
-    static let menuSelectedItemText = UIColor(named: "menuSelectedItemText") ?? UIColor.black
-    static let menuUnselectedItemText = UIColor(named: "menuUnselectedItemText") ?? UIColor.black
-    static let loadingIcon = UIColor(named: "loadingIcon") ?? UIColor.black
-    static let avatarRing = UIColor(named: "avatarRing") ?? UIColor.black
-    static let networkAnimation = UIColor(named: "networkAnimation") ?? UIColor.black
-    static let selectedTab = UIColor(named: "selectedTab")!
-    static let unselectedTab = UIColor(named: "unselectedTab")!
-    static let textInputBorder = UIColor(named: "textInputBorder")!
-    static let textInputBackground = UIColor(named: "textInputBackground")!
+    static let appBackground = UIColor(named: "appBackground", in: Bundle.current, compatibleWith: nil) ?? UIColor.white
```

Comment:
This gets these working in the test target. In the future I would like to generate this file from an .xcassets directory with SwiftGen so I didn't worry about finding a cleaner solution.